src/dbml.pegjs

```diff
@@ -85,7 +88,10 @@ DoubleQuotedString = '"' content:($[^"\\]+ / '\\"' { return '"' } / [\\])* "'" {
 
 Function = '`' [^`]* '`' { return text(); }
 
-Comment = _ "//" _ comment:LineOfText { return { comment }; }
+Comment = SingleLineComment / MultiLineComment
+SingleLineComment = _ "//" _ comment:LineOfText { return { comment }; }
+MultiLineComment = "/*" comment:(("*/" { return ""; }) / MultiLineCommentContent) { return comment; }
+MultiLineCommentContent = head:. tail:(!"*/" c:. { return c; })* "*/" { return [head, ...tail].join(""); }
```

Comment:
Do you mind adding a new example that covers multiline comments?

Reply:
See `advanced_dbml.dbml`

Reply:
Thanks!